portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -34,17 +37,22 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  String comNum =  "5";
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    Query query = new Query("comment");
-    
+
+   int comLimit = Integer.parseInt(comNum);
```

Comment:
Do we need to handle the error case where "comNum" is not a string representing an integer?

Reply:
Note that it will throw a NumberFormatException in error case (see https://docs.oracle.com/javase/7/docs/api/java/lang/Integer.html#parseInt(java.lang.String).

Reply:
I don't think that would happen because when the getParameter function is called with input "number-comments" it will always return a string representing an integer and for cases when the user does not specify, the default string is "5". What do you think?